### PR TITLE
Overabundance added

### PR DESCRIPTION
**Pull request by elliott-king**
NOTE: Does not work with Mishra's Workshop or Eldrazi Temple (BUT neither does Mana Flare). There exists a problem with the game logic.

See my open issue [here](https://github.com/magefree/mage/issues/2826).

### CODE REVIEW

File: Mage.Sets/src/mage/cards/o/Overabundance.java [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+package mage.cards.o;
+
+import java.util.UUID;
+import mage.cards.CardImpl;
+import mage.cards.CardSetInfo;
+import mage.constants.CardType;
+
+/**
+ *
+ * @author anonymous
```
Comment (spjspj) [inline Mage.Sets/src/mage/cards/o/Overabundance.java]: You can add your moniker to replace anonymous.  There's a file in the same directory as gen-card.pl for your signature.

File: Mage.Sets/src/mage/cards/o/Overabundance.java [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+ */
+public class Overabundance extends CardImpl {
+
+    public Overabundance(UUID ownerId, CardSetInfo setInfo) {
+        super(ownerId, setInfo, new CardType[]{CardType.ENCHANTMENT}, "{1}{R}{G}");
+        
+
+        // Whenever a player taps a land for mana, that player adds one mana to his or her mana pool of any type that land produced, and Overabundance deals 1 damage to him or her.
```
Comment (spjspj) [inline Mage.Sets/src/mage/cards/o/Overabundance.java]: Hi, was there a commit that didn't get added into the pull request?

Comment (elliott-king) [inline Mage.Sets/src/mage/cards/o/Overabundance.java]: Thanks, my bad!

**Comment by LevelX2** [comment]
Thanks for your contribution.

**Pull request merged**